groups_in_group/groups_in_group.tf

```diff
@@ -0,0 +1,28 @@
+terraform {
+  required_providers {
+    gsuite = {
+      source  = "DeviaVir/gsuite"
+      version = "0.1.58"
+    }
+  }
+}
+
+variable "groups" {
+  type        = map(any)
+  description = "contains objects representing all defined Google Groups"
+}
+
+variable "group" {
+  type        = any
+  description = "contains an object representing a Google Group"
+}
+
+/**
+ * Ressource documentation: https://registry.terraform.io/providers/DeviaVir/gsuite/latest/docs/resources/group
+ */
+
+resource "gsuite_group_member" "member" {
+  for_each   = toset(var.group.groups)
+  group      = var.group.email
+  email      = var.groups[each.value].email
+}
```

Comment:
add newline at the end of file, should be part of .editorconfig

Reply:
yes sure, it is part of the config. editorconfig extension was not active in my ide. will be fixed with the next adaption.